doc/source/operating.rst

```diff
@@ -263,4 +263,50 @@ your file system). The procedure to restore is no more complicated than initial
 deployment: restore your index and storage backups, reinstall Gnocchi if
 necessary, and restart it.
 
+How to clear Gnocchi data
+=========================
+
+If you ever want to start fresh or need to clean Gnocchi data, this can be
+easily done. You need to clean the measures (incoming), aggregates (storage)
+and indexer data storage.
+
+Once that is done, if you want to re-initialize Gnocchi, you need to call
+`gnocchi-upgrade` so it re-initialize the different drivers.
```

Comment:
i'm fairly certain you'd need to reset storage sacks. it only sets it if the file is missing: https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/incoming/__init__.py#L68

Reply:
Good catch, added for a few incoming backends.